orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -1,386 +1,425 @@
 {
```

Comment:
<p>First lines, has capital V; please take use small v. I believe this is a scalar so it can stay italicised.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>